dist/.magento.env.yaml

```diff
@@ -262,6 +262,20 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTIC_SUITE_CONFIGURATION - use this variable to retain customized ElasticSearch service settings                 #
```

Comment:
please describe in what array element these configs will be set

Reply:
Fixed

Reply:
Please explain

Reply:
All data which customer set to this config - added to system/default/smile_elasticsuite_core_base_settings, right?
and there may be mush more configs than you added to examples, so, need to add info in what format they may add these configs. Something like: you may add/extend all configurations that are stored under path xxx for ElasticSuite settings

Reply:
And add info that that engine should be changed in SEARCH_CONFIGURATION

Reply:
Fixed

Reply:
Do not see mentions about SEARCH_CONFIGURATION

Reply:
We change engine automatically